src/util/semantic-versioning.ts

```diff
@@ -0,0 +1,23 @@
+export class SemanticVersioning {
+    private version: string;
+
+    constructor(version: string) {
+        this.version = version;
+    }
```

Comment:
nit: Can be simplified to the following.
```suggestion
    constructor(private version: string) {
    }
```

Reply:
Unfortunately this doesn't work. I tried it out as i wasn't sure but it throws a duplicate identifier compile error.